danger.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "git", "~> 1"
   spec.add_runtime_dependency "colored", "~> 1.2"
   spec.add_runtime_dependency "faraday", "~> 0"
+  spec.add_runtime_dependency "faraday-http-cache", "~> 1.0"
```

Comment:
For our use case, wouldn't it be enough to just cache the GitHub JSON response ourself?